### DISCUSSION TITLE
LAB 2.04 Task 2

### DISCUSSION THREAD

**Discussion by moralesluis26** [rramosp/ai4eng.v1#135] [02 - PYTHON]
Saludos profesor y monitor.
Resulta que he tratado de resolver este ejercicio pero aún no doy con la forma de hacerlo, he intentado con la función del task 1 (make_heaps) y también con el "for" pero aún no lo logro. Les agradecería si me indican una función con la cual pueda realizar este task, le agradezco de antemano.

**Comment by Felipe-RA** [comment]
Saludos moralesluis

Muestranos qué haz probado para poderte guiar mejor

**Reply by moralesluis26** [reply]
Saludos Felipe-Ra.
Ya lo pude solucionar, gracias. Adjunto imágen del código al que llegué.
![imagen](https://user-images.githubusercontent.com/46792756/172724330-96a5d7d4-0029-4732-8e9a-2df12a34c9ee.png)
